xdp-loader/xdp-loader.c

```diff
@@ -294,6 +313,7 @@ int do_help(__unused const void *cfg, __unused const char *pin_root_path)
 		"       load        - load an XDP program on an interface\n"
 		"       unload      - unload an XDP program from an interface\n"
 		"       status      - show current XDP program status\n"
+		"       collect     - clean up dead links in XDP bpffs directory\n"
```

Comment:
Let's just name the action 'clean' as well - and don't forget to update the man page (README.org) as well